Makefile

```diff
@@ -19,32 +19,37 @@ DOCKER_TAG = unstable
 DOCKER_IMAGE = cosmos/ethermint
 ETHERMINT_DAEMON_BINARY = emintd
 ETHERMINT_CLI_BINARY = emintcli
+GO_MOD=GO111MODULE=on
```

Comment:
It should be possible to simplify this to `GO111MODULE=on`, with the additions below removed. Once its declared here it is within the scope of all the commands this file calls (eg. `go build`).

Reply:
Tried this, doesn't work when GO111MODULE=on is not exported and code is run within the $GOPATH. I ran into this issue before and was very picky about being included before every go command which required the dependencies.

Reply:
It seems to work for me if I just make this change:
```
GO111MODULE=on
GO_MOD=
```
Any chance you have `GO111MODULE` set on your machine already? Maybe in `.bashrc` or `.profile`?

Reply:
Doesn't work to just do this when run in $GOPATH, although is sufficient for running code outside of $GOPATH.